tests/desktop/consumer_pages/test_details_page.py

```diff
@@ -73,7 +73,6 @@ def test_that_reports_abuse_as_anonymous_user(self, mozwebqa):
         report_abuse_box = details_page.click_report_abuse_button()
 
         Assert.true(report_abuse_box.is_visible)
-        Assert.false(report_abuse_box.is_report_button_enabled)
```

Comment:
The button is always enabled, we just get an error message if we don't enter any text in the box